deepchem/models/torch_models/progressive_multitask.py

```diff
@@ -0,0 +1,232 @@
+import torch
+import torch.nn as nn
+
+from deepchem.utils.typing import OneOrMany
+
+from collections.abc import Sequence as SequenceCollection
+from typing import List
+
+
+class ProgressiveMultitask(nn.Module):
```

Comment:
Add this to the docs

Reply:
Usually, only the TorchModel instances of the models are added to the docs. eg. SeqtoSeq and DMPNN. Can I add the ProgressiveMultitaskRegressor and ProgressiveMultitaskClassifier to the docs in the following PRs instead of this?